example.html

```diff
@@ -2,6 +2,9 @@
 <html>
 <head>
 	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta content="http://branch.io/img/logo_icon_black.png" property="og:image" />
+	<meta content="Branch Metrics Web SDK Example App" property="og:title" />
+	<meta content="A basic example to demonstrate some of the ways that the Web SDK can be used" property="og:description" />
 	<title>Branch Metrics Web SDK Example App</title>
```

Comment:
These extra tags are to show how the data can be pulled from the head if they are left out of calls